modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -795,15 +804,54 @@ Ransomware Protection (Enterprise Edition onyl)
 Marketplace URL: https://marketplace.owncloud.com/apps/ransomware_protection[Ransomware Protection]
 
 Use these commands to help users recover from a Ransomware attack. 
-You can find more information about the application in the documentation <../../enterprise/ransomware-protection/index>.
+You can find more information about the application in the documentation 
+<../../enterprise/ransomware-protection/index>.
 
-[width="100%",cols="50%,82%",]
+....
+sudo -u www-data occ ransomguard:scan <timestamp> <user>
+....
+
+Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `<timestamp>` +
```

Comment:
Curiously, why are `<timestamp>` and `<user>` on separate lines? They'd work better being on the same line.

Reply:
Because it is easier to read. They have both the same description text therefore in the same row but separated in a new line.